components/compositing/Cargo.toml

```diff
@@ -13,10 +13,11 @@ path = "lib.rs"
 
 [features]
 default = []
-gl = ["gleam", "pixels"]
```

Comment:
I think you can also revert this change now too.

Reply:
done in 36dd2a70836b5b7253c7cb782ca41a755fcf7abf